core/src/main/scala/org/apache/spark/sql/delta/DeltaSharedExceptions.scala

```diff
@@ -45,3 +45,16 @@ class DeltaUnsupportedOperationException(
     with DeltaThrowable {
     override def getErrorClass: String = errorClass
 }
+
+object DeltaAnalysisException {
+  def apply(
+      errorClass: String,
+      messageParameters: Array[String],
+      cause: Option[Throwable] = None): AnalysisException = {
+    new AnalysisException(
```

Comment:
Did we run the integration test we added?

Reply:
I had to update the integration test. The test update is part of the PR. Check [here](https://github.com/delta-io/delta/pull/1227/files#diff-a78f03ac6f31044a96ad3a4c26ea57b61d61ca62757f80550be5f0bd35e24808R52).